pkg/controller/endpointmonitor/endpointmonitor_controller.go

```diff
@@ -129,18 +137,17 @@ func (r *ReconcileEndpointMonitor) Reconcile(request reconcile.Request) (reconci
 				return reconcile.Result{RequeueAfter: delay}, nil
 			}
 			err = r.handleCreate(request, instance, monitorName, r.monitorServices[index])
+			if err != nil {
+				log.Errorf("Error while handling create: %s", err)
+				return reconcile.Result{RequeueAfter: RequeueTime}, err
+			}
```

Comment:
This is redundant since it already requeue's after create or update action:
https://github.com/stakater/IngressMonitorController/blob/master/pkg/controller/endpointmonitor/endpointmonitor_controller.go#L135

Reply:
Thanks, I'll fix that.